src/lib/falcor/FalcorRouter.js

```diff
@@ -1,53 +1,55 @@
 import BaseRouter from "falcor-router"
 import { ghostArticleQuery } from 'lib/ghostAPI'
-import { sqlArticleQuery, sqlAuthorQuery } from 'lib/sql'
+import { dbAuthorQuery, dbArticleQuery, dbAuthorArticleQuery, dbInfoPagesQuery, dbArticleIssueQuery,
+dbArticleAuthorQuery, dbLatestIssueQuery, dbCategoryQuery, dbCategoryArticleQuery,
+dbFeaturedArticleQuery, dbEditorPickQuery, dbIssueCategoryQuery, dbIssueQuery } from 'lib/mariaDB'
 import falcor from 'falcor'
+import _ from 'lodash';
 
-const $ref = falcor.Model.ref; 
+const $ref = falcor.Model.ref;
 
-// FLAG TO CHECK IF YOU WANT TO ACCESS GHOST API AND DATABASES
-// THIS IS JUST FOR DEVELOPMENT WHILE WE DON'T HAVE ACTUAL SERVERS SET UP
-const USE_DATABASES = false; 
-
-// Transform ghost API names into our names
-const mapGhostNames = (name) => {
-  switch(name) {
-    case 'pubDate':
-      return 'published_at';
-    case 'body':
-      return 'markdown';
+let mapGhostNames = (falcorName) => {
```

Comment:
Well that's also what I was trying to. 

It depends a bit on @zanemountcastle because the reason I kept it now was because teaser is a lot more clear than "meta_description" but maybe we should just use that.

I got rid of all the others and changed the names accordingly